spec/support/vcr.rb

```diff
@@ -9,6 +9,9 @@
   # set to false when recording new stubs, true otherwise
   #
   config.ignore_request { |req| URI(req.uri).path.eql?('/oauth/token') }
+
+  # ignore the webdriver session
+  config.ignore_hosts '127.0.0.1'
```

Comment:
can this be more specific to a request - [CCCD did a similar thing to ignore capybara](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/master/spec/vcr_helper.rb#L23-L42) - otherwise it could affect ability to record cassettes against local api